components/dropdown/demo/api.min.js

```diff
@@ -9,28 +10,27 @@ function showExample() {
 function hideExample() {
   const btn = document.querySelector('#hideExampleBtn');
   const dropdown = document.querySelector('#hideExample');
+
   btn.addEventListener('click', () => {
     dropdown.hide();
   });
 }
 
 /* eslint-disable jsdoc/require-jsdoc, no-magic-numbers */
-function initExamples(initialCount = 0) {
-  const MAX_RETRY_ATTEMPTS = 20;
```

Comment:
Is there a reason that you removed these global variables?

Reply:
this is a script generated file.